src/utils/helper-functions/typecheck.ts

```diff
@@ -1,50 +0,0 @@
-import type { TSESLint, TSESTree } from '@typescript-eslint/experimental-utils'
```

Comment:
This file was basically a copy from https://github.com/cartant/eslint-etc/blob/02584e770982afca432b9feb2c457483589dcde5/source/get-type-services.ts... so instead of coyping it here, why not use it from `eslint-etc` and avoid the dependency on `tsutils-etc` explicitly (yes, it still there, because `eslint-etc` depends on it, but...)

Reply:
I think I might have copied it when it wasn't a part of the public api 🤔
Either way, good catch.